md/bonita-bpm-portal-interface-overview.md

```diff
@@ -1,32 +1,49 @@
 
-The **User** is responsible of performing the tasks for which she is a candidate and also for starting new cases of the processes to which she has access.
+In Bonita 7.12, Bonita Applications offer:
+  - the Bonita Administrator Application
+  - the Bonita User Application
+  
+Both Bonita Applications can be downloaded from the Welcome Page of the Studio, in the "Resources" tile.
 
-Example of the User profile interface
-![](images/user_tasklist.png)<!--{.img-responsive}-->
+Applications are more flexible than the portal, in two ways:
+   - The navigation can be customized: remove the pages that are not needed, add custom pages that are needed. This is the equivalent of Custom Profiles, but with the flexibility of an application. 
+   - Some pages of those applications have been recreated with Bonita UI Designer (see the list [here]). This means that you can open and customize the page to make it fit the precise needs of the users. Starting with an existing page accounts for a big gain of time.
 
-To know more about the use and optimization of the task list, go to [User task list](user-task-list.md).
+Moreover, for Bonitasoft as for its users, applications are free from the Google Web Toolkit framework used to build an important part of Bonita Portal, which accounts for an important technical update and optimized maintenance ahead of us.
 
-## Access to menus
+## Custom User Interfaces
 
-Each profile has access to a different set of single page menus or drop down menus, containing filters and action buttons related to their user rights.
+You can also create views that fit users' needs better than the provided User Interfaces.  
+In Bonita Portal, this has been made possible through the concept of [custom profiles](custom-profiles.md), new Portal views *made of Bonita pages and/or custom pages*.  
+Then, with Bonita 7.0, Living applications have given the possibility to create applications *made of custom pages*, with their own look & feel. The pages can be created with Bonita UI Designer or in another environment.  
+With Bonita 7.12 and the availability of Administrator and User applications, Living applications can be *made of Bonita pages and/or custom pages* too.
 
 ::: info
-**:fa-info-circle:Note**: The tenant administrator, logged as the Technical user, needs to map the Administrator profile with individuals in the company (or groups, or roles) to let the Administrator(s) manage the rest of the organization and other users. See [First steps after setup/Create an Administrator user](first-steps-after-setup.md).
-:::
-
-See also [Profiles overview](profiles-overview.md).  
-See also [Custom profiles](custom-profiles.md).
+**Note:** As Portal will be replaced by applications in next versions of Bonita, we strongly encourage you **not** to create Custom profiles for Portal views anymore, but applications.  
```

Comment:
we strongly encourage you **not** to create -> we strongly recommend, you can't quite encourage to not do something :)

Reply:
this is not up to date